src/main/java/codeu/model/store/basic/MessageStore.java

```diff
@@ -89,6 +97,22 @@ public void addMessage(Message message) {
     persistentStorageAgent.writeThrough(message);
   }
 
+  /** Delete a message from the current set of messages known to the application */
+  public void deleteMessage(Message message) {
+    messages.remove(message);
+    persistentStorageAgent.delete(message);
+  }
+
+  /** Delete an old message sent by this User */
+  public void deleteOldMessages(UUID userId) {
```

Comment:
Is this method supposed to be deleting multiple messages or a single message? The method comment and method name seem to suggest different things.

I'd probably supply a number to this method with the number of messages that need to be deleted instead of having to call this method multiple times.